selinux/osbuild.te

```diff
@@ -0,0 +1,72 @@
+policy_module(osbuild, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+attribute_role osbuild_roles;
+roleattribute system_r osbuild_roles;
+
+type osbuild_t;
+type osbuild_exec_t;
+application_domain(osbuild_t, osbuild_exec_t)
+role osbuild_roles types osbuild_t;
+
+########################################
+#
+# osbuild local policy
+#
+
+allow osbuild_t self:fifo_file manage_fifo_file_perms;
+allow osbuild_t self:unix_stream_socket create_stream_socket_perms;
+
+domain_use_interactive_fds(osbuild_t)
+
+files_read_etc_files(osbuild_t)
```

Comment:
This rule is not required any more. We allowed in base policy.

Reply:
Done